docs/output-targets/dist.md

```diff
@@ -67,10 +69,8 @@ import * as utils from '@utils';
 import * as utils from '../path/to/utils';
 ```
 
-This flag defaults to `false` when omitted from a Stencil configuration file.
-
 :::tip
-If using the `dist-collection` output target directly, the same result can be achieved using the `transformAliasedImportPaths` flag on the target's config.
+If using the `dist-collection` output target directly, the same result can be achieved using the [`transformAliasedImportPaths`](../config/01-overview.md#transformaliasedimportpaths) flag on the target's config.
```

Comment:
This link points to the top-level config flag not the one for the `dist-collection` target specifically.

Reply:
👍 4090dda43